kolibri/core/templatetags/kolibri_tags.py

```diff
@@ -91,41 +91,27 @@ def kolibri_set_server_time():
 
 @register.simple_tag(takes_context=True)
 def kolibri_bootstrap_model(context, base_name, api_resource, **kwargs):
-    # Don't bootstrap when the debug panel is active, because it prevents profiling
-    if not getattr(settings, 'ENABLE_DATA_BOOTSTRAPPING'):
-        return ''
-    # check necessary for when there is no initial content databases
-    if 'kwargs_channel_id' in kwargs:
-        if not context['currentChannel']:
-            return ''
     response, kwargs, url_params = _kolibri_bootstrap_helper(context, base_name, api_resource, 'detail', **kwargs)
     html = ("<script type='text/javascript'>"
-            "var model = {0}.resources.{1}.createModel(JSON.parse({2}), {3});"
+            "var model = {0}.resources.{1}.createModel({2}, {3});"
```

Comment:
Does rendering with `response.data` make it so we don't need `JSON.parse` anymore?

Reply:
Yes, the JSONRenderer renders it as valid JSON (which is a subset of valid Javascript) so it doesn't need to be parsed, as long as we make sure the utf decoding is there!

Reply:
It might be marginally safer to run JSON.parse on it, to prevent against injection attacks though.

Reply:
but how are we open to injection attacks if we are the ones passing the data to these functions?

Reply:
If someone entered a valid javascript object method as the data of a model text field, which, unless we do sanitization on all text fields to prevent javascript being put into them, is possibly possible.

Reply:
(it's unlikely, but it's possible)